### PR TITLE
RF|_NewItemTest->testTC_126_001_OKD_NewItemPageFormSubmission

### PR DESCRIPTION
**Pull request by marolespb8**
Test TC_126_001 was moved to _NewItemTest, class OKDisableTest was deleted.

### CODE REVIEW

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/_NewItemTest.java [hunk 1]
```diff
@@ -84,4 +84,17 @@ public void testCopyDataFromExistingItemPositive() {
         asserts.assertEquals(getDriver().findElement(By.name(GITHUB_URL)).getAttribute("value"), URL_INPUT);
         asserts.assertAll();
     }
+
+    @Test
+    public void testTC_126_001_OKD() {
+
```
Comment (zenicko) [inline src/test/java/_NewItemTest.java:90]: Delete the line

File: src/test/java/_NewItemTest.java [hunk 2]
```diff
@@ -84,4 +84,17 @@ public void testCopyDataFromExistingItemPositive() {
...
+        String expectedError = "» This field cannot be empty, please enter a valid name";
```
Comment (zenicko) [inline src/test/java/_NewItemTest.java:91]: final String ...

File: src/test/java/_NewItemTest.java [hunk 3]
```diff
@@ -84,4 +84,17 @@ public void testCopyDataFromExistingItemPositive() {
...
+
+        getDriver().findElement(By.className("task-link-text")).click();
+        getDriver().findElement(By.cssSelector("div.btn-decorator")).click();
+        String actualError = getDriver().findElement(
+                By.cssSelector("#itemname-required.input-validation-message")).getText();
+
+        Assert.assertEquals(actualError, expectedError);
+    }
 }
```
Comment (zenicko) [inline src/test/java/_NewItemTest.java:100]: Add the empty line

**Review by zenicko — APPROVED** [review]
Some ideas to make the code clearly

**Comment by marolespb8** [comment]
Thank you.

**Comment by marolespb8** [comment]
My test is gone in _NewItemTest.

**Pull request closed without merging**